files/firmware_status.py

```diff
@@ -104,6 +108,8 @@
 
 jverlist = json.loads(pr.stdout)
 for verd in jverlist:
+    if verd['version'].endswith('b'):
```

Comment:
```python
    if ignore_beta and verd['version'].endswith('b'):
```

Reply:
@crpb 
sorry, my merge was some kind of hasty, can you please fix the condition?